lhd_app/lib/screens/requestaquote.dart

```diff
@@ -0,0 +1,2457 @@
+                    '\n\n' +
+                    'Also, give some serious consideration to the exterior appearance of your home. It usually does not take much to add a little design and totally change a plain exterior into a unique, eye-catching design. Ask your designer for suggestions and some preliminary drawings.' +
+                    '\n\n' +
+                    'Because your home will be totally unique to your needs and wants, house style, site location, etc., it will be impossible to completely cover every area or aspect in this guide. However, it is my hope that it will get your creative juices flowing and be a blessing to you in sparking constructive dialogue as you work through the design stages of your new home.' +
+                    '\n\n' +
+                    'One last design tip before you jump into it feet first: Don’t let yourself be limited to just answering the questions and sketching out a few things.' +
+                    '\n\n' +
+                    'Take the time to make notes – where applicable – about the reasons why you answered the question in the way you did. Not only will it clarify or even change your initial thoughts, but it’ll help your designer better understand why you want something. And just maybe give him practical insight about your needs and desires to work with you towards an even better solution.' +
+                    '\n\n',
+                textAlign: TextAlign.justify, // Align text justify
+              ),
+            ),
+            Text(
+                'Want to start planning your dream house? Request a quote now! \n',
+                style: TextStyle(fontStyle: FontStyle.italic)),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.only(
+                      right: 15.0, bottom: 10.0), // Add margin to the right
+                  child: ElevatedButton(
+                    style: ElevatedButton.styleFrom(
+                      backgroundColor: accent2,
+                    ),
+                    onPressed: () {
+                      Navigator.push(
+                        context,
+                        MaterialPageRoute(
+                          builder: (context) => Basement(),
+                        ),
+                      );
+                    },
+                    child: Text(
+                      'Request a Quote',
+                      style: TextStyle(color: Colors.white),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+
+
+
+
+
+
+
+
+
+//BASEMENT DESIGN
+class Basement extends StatefulWidget {
```

Comment:
You may create a folder for the stages and separate every pages there or you may look for the pub.dev if they have library for progress form